R-package/README.md

```diff
@@ -128,6 +128,14 @@ If you want to force `LightGBM` to use MSYS2 (for any R version), pass `--use-ms
 Rscript build_r.R --use-msys2
 ```
 
+**MinGW**
```

Comment:
Users still may want to compile with MinGW. Not one bundled in Rtools, but with a normally installed one.